src/Steeltoe.Initializr.WebApp/ClientApp/src/components/Home.js

```diff
@@ -68,26 +68,21 @@ export class Home extends Component {
         this.setState(prevState => ({showMore: !prevState.showMore}))
     }
     handleInputChange(name, selectedValue) {
-        this.setState({
-            [name]: selectedValue,
-            "steeltoeVersionInvalid": ""
-        });
-        // if (name === "targetFrameworkVersion" && selectedValue === "netcoreapp3.1" && this.state.steeltoeVersion === "2.3.0") {
-        //     this.setState({
-        //         "steeltoeVersion": "2.4.0",
-        //         [name]: selectedValue,
-        //         "steeltoeVersionInvalid": ""
-        //     })
-        // }
-        // else if(name === "steeltoeVersion" && selectedValue === "2.3.0" && this.state.targetFrameworkVersion === "netcoreapp3.1"){
-        //     this.setState({
-        //         "steeltoeVersion": "2.4.0",
-        //         "steeltoeVersionInvalid": "2.4.0 is the lowest version compatible with netcoreapp3.1",
-        //     })
-        // }
-        // else {
-        // ....
-        // }
+        if (name === "steeltoeVersion" && selectedValue === "3.0.0-m3" && this.state.targetFrameworkVersion === "netcoreapp2.1") {
+            this.setState({
+                "steeltoeVersionInvalid": "3.0.0-m3 is not compatible with netcoreapp2.1",
```

Comment:
In this case, where the user selects Steeltoe version 3.0.0, the message should be more like " Steeltoe 3.x.x is only compatible with netcoreapp3.1 and above."